src/AspNetCore/Inc/inprocessapplication.h

```diff
@@ -53,8 +54,14 @@ class IN_PROCESS_APPLICATION : public APPLICATION
 
     HRESULT
     LoadManagedApplication(
-            VOID
-        );
+        VOID
+    );
+    
+    HRESULT
+    OnClientDisconnect(
+        IHttpContext* pHttpContext,
+        BOOL fClientDisconnected
```

Comment:
why do we need thisfClientDisconnected?  We can assume client is disconnected when this function is called